apps/api/src/app/auth/services/passport/jwt.strategy.ts

```diff
@@ -20,12 +20,6 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
       throw new UnauthorizedException();
     }
 
-    this.logger.assign({
-      userId: user._id,
-      environmentId: payload.environmentId,
-      organizationId: payload.organizationId,
-    });
-
```

Comment:
This is now handled in a single place in the Auth Guard, for all Auth strategies.